tools/mo/openvino/tools/mo/utils/get_ov_update_message.py

```diff
@@ -32,3 +32,12 @@ def get_tf_fe_message():
               'Find more information about new TensorFlow Frontend at {}'.format(link)
 
     return message
+
+
+def get_compression_message():
+    link = "https://docs.openvino.ai/latest/openvino_docs_MO_DG_FP16_Compression.html"
+    message = '[ INFO ] --compress_to_fp16 argument was not specified explicitly. Compressed FP16 IR will be generated by default.\n' \
+              'If you get lower accuracy, please consider disabling compression explicitly ' \
+              'by adding argument --compress_to_fp16=False.\n' \
+              'Find more information about compression to FP16 at {}'.format(link)
+    return message
```

Comment:
please prepare unit-test for this behaviour like here: https://github.com/openvinotoolkit/openvino/blob/master/tools/mo/unit_tests/moc_tf_fe/check_info_messages.py

Reply:
done